definitions/infra-vsphereresourcepool/golden_metrics.yml

```diff
@@ -54,3 +54,17 @@ memoryFree:
       from: VSphereResourcePoolSample
       eventId: entityGuid
       eventName: entityName
+vmCount:
+  query:
+    eventId: entityGuid
+    select: latest(`vmCount`)
+    from: VSphereResourcePoolSample
+  unit: COUNT
+  title: Virtual machines count
+memSwapped:
+  query:
+    eventId: entityGuid
+    select: (average(`mem.swapped`)) * 1024 * 1204
```

Comment:
given that the unit is BYTES, is this `* 1024 * 1024` actually correct?

Reply:
It makes sense, this query was copied from https://github.com/newrelic/entity-definitions/blob/c153c727ec0aa87543018927885636082ae7f348/definitions/infra-vsphereresourcepool/summary_metrics.yml#L11 

After some investigation, this query  shows https://onenr.io/0BQrK3JoNwZ
select (average(`mem.free`)) * 1024, (average(`mem.free`)) * 1024 * 1024 FROM VSphereResourcePoolSample
4.83 G
4.94 T -> it seems mistaken